tests/topotests/bgp_vpnv4_ebgp_vpn_auto/test_bgp_vpnv4_ebgp.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+
+#
+# test_bgp_vpnv4_ebgp.py
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2022 by 6WIND
```

Comment:
is this the correct copyright statement?

Reply:
Changed the copyright appropriately